### PR TITLE
✨ Add well known status rule for daemonsets

### PR DESCRIPTION
**Pull request by zhujian7**
## Related issue(s)

Fixes #

### CODE REVIEW

File: test/integration/cloudevents/statusfeedback_test.go [hunk 1]
```diff
@@ -541,4 +541,186 @@ var _ = ginkgo.Describe("ManifestWork Status Feedback", func() {
 			}, eventuallyTimeout, eventuallyInterval).ShouldNot(gomega.HaveOccurred())
 		})
 	})
+
+	ginkgo.Context("DaemonSet Status feedback", func() {
```
Comment (qiujian16) [inline test/integration/cloudevents/statusfeedback_test.go:545]: @skeeey why do we still need to create separated integration test case for cloudevent?

**Review by qiujian16 — COMMENTED** [review]
/approve
/lgtm

File: test/integration/cloudevents/statusfeedback_test.go (hunk 1 above)
Comment (skeeey) [reply test/integration/cloudevents/statusfeedback_test.go:545]: the could events supports single manifest, for single manifest, some integration cannot pass

**Pull request merged**